src/Allocator/Process.hs

```diff
@@ -0,0 +1,22 @@
+module Allocator.Process where
+
+import Foundation
+import Manager
+import Control.Effect.State
+
+newtype Mem = Mem [Page 'Ram] deriving Eq
```

Comment:
I'm thinking now, that we don't need this type at all. It doesn't carry any useful information, nor it describes something new. We already have `PageTable` in `State` effect, which holds our Ram AND Swap, so, maybe we can get rid of this.

Reply:
We've already discussed this recently and I though that we came to the decision that in the first case `PageTable` is the abstraction over `Ram` and `Swap` pages in the manager and `Mem` is the abstraction over allocated Ram pages. Notice that allocator doesn't know anything about page location (swap or ram), because it's all ram for him.